aqua-tests/.gitignore

```diff
@@ -0,0 +1,8 @@
+.idea
+.DS_Store
+**/node_modules
+**/target/
+.repl_history
+.vscode/settings.json
+src/ts/src/aqua
+src/js/src/aqua
```

Comment:
```suggestion
src/js/src/aqua

```

Reply:
this is autogenerated gitignore I can remove unncecessary stuff from there

the .fluence dir is needed because I decided to generate secret keys before hand instead of generating them on the fly which also is not as convenient with fluence cli, because it doesn't allow passing secret key as a flag because it's a bad practice anyway

Reply:
To run tests in parallel you should use different init peer ids (and different keypairs)

Reply:
Maybe it's good to generate random labels for resources, and only one keypair will be enough